src/components/ToDoList/ToDoList.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+}
+
+export const ToDoList: React.FC<Props> = ({todos}) => {
+  return (
+    <section className="todoapp__main">
+      {todos.map((todo) => (
+      <div className="todo">
+      <label className="todo__status-label">
+        <input type="checkbox" className="todo__status" />
+      </label>
+
+      <span className="todo__title">{todo.title}</span>
+      <button type="button" className="todo__remove">×</button>
+
+      {/* 'is-active' class puts this modal on top of the todo */}
+      <div className="modal overlay is-active">
+        <div className="modal-background has-background-white-ter" />
+        <div className="loader" />
+      </div>
+    </div>
+    ))}      
```

Comment:
don't forget about code style
```suggestion
      {todos.map((todo) => (
        <div className="todo">
        <label className="todo__status-label">
          <input type="checkbox" className="todo__status" />
        </label>

        <span className="todo__title">{todo.title}</span>
        <button type="button" className="todo__remove">×</button>

        {/* 'is-active' class puts this modal on top of the todo */}
        <div className="modal overlay is-active">
          <div className="modal-background has-background-white-ter" />
          <div className="loader" />
        </div>
      </div>
    ))}      
```